src/Server/Player/AppleLogic.py

```diff
@@ -48,15 +49,21 @@ def _calculate_attack_force(self, x, y):
         return attack_x, attack_y
 
     def _attack(self, force):
-        if super()._attack() and not self._is_flying:
+        if self._can_attack and not self._is_flying:
+            super()._attack()
             self._is_attacking = True
             self._add_force(force)
             self._impact()
+
             return True
+        else:
+            print("you cant attack", self._can_attack)
         return False
```

Comment:
Where is the return value checked?! - I didn't find the place.
If it isn't checked, why False and True values - misleading.

Reply:
See OrangeLogic review comment - consider removing or improving print message.